### DISCUSSION TITLE
How to remove anchor points from specific glyphs only and not entire font?

### DISCUSSION THREAD

**Discussion by nycionx** [fontforge/fontforge#5725] [General]
How to remove anchor points from specific glyphs only and not the entire font?
Is there a way to do this scripts wise (ff lang)? 
I know how to remove ALL anchor points from the entire font, but I need it from specific glyphs only.
Thanks

**Comment by mdpugh** [comment]
This worked for my purposes:


```
# Function to remove an anchor point by its class name
def removeAnchor(glyph, anchor_class):
    # Copy the anchorPoints list
    anchor_points_copy = list(glyph.anchorPoints)
    
    # Search for the anchor points with the matching class name and remove them
    anchor_points_copy = [anchor for anchor in anchor_points_copy if anchor[0] != anchor_class]
    
    # Copy the modified list back into the glyph's anchorPoints
    glyph.anchorPoints = anchor_points_copy
```


Of course, you can search on other attributes too.  The idea is to copy the anchor point into a list, edit the list, copy the list back into anchorPoints.  If there is a simpler way, I have not found it.
